responsemanager/server.go

```diff
@@ -108,6 +129,16 @@ func (rm *ResponseManager) abortRequest(p peer.ID, requestID graphsync.RequestID
 		return errors.New("could not find request")
 	}
 
+	_, span := otel.Tracer("graphsync").Start(trace.ContextWithSpan(rm.ctx, response.span), "abortRequest")
+	defer span.End()
+	span.RecordError(err)
+	span.SetStatus(codes.Error, err.Error())
+	if response.span.IsRecording() {
```

Comment:
Why are we checking IsRecording() here but not elsewhere?

Reply:
Because this is the only place I've observed multiple calls coming in - e.g. in the NetworkDisconnect test, the receiver can have a call come through `abortRequest()` between 1 and 3 times.

I don't know whether closing a span that's already been closed makes any difference or not, this `if` was a late addition and not because I had any errors or weirdness, I suspect it's a noop underneath. :shrug: whether I can clean this up with your new work.